comp/metadata/packagesigning/utils/packagesignigutils.go

```diff
@@ -0,0 +1,109 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2016-present Datadog, Inc.
+
+// Package utils defines shared methods in package signing component
+package utils
```

Comment:
Looks like this utils functions are meant to be use for Linux. Should we rename the file to `comp/metadata/packagesigning/utils/linux.go`?

Reply:
the package signing is normally ment for linux only, this is not necessarily a `common` though. I can certainly rename it `linux.go`